Testscripts/Linux/SetupRDMA.sh

```diff
@@ -606,8 +606,23 @@ function Main() {
 		Verify_File $benchmark_bin
 	fi
 
+    HOMEDIR=$(pwd)
+	cd ~
+	LogMsg "Proceeding OSU MPI Benchmark test installation"
+	LogMsg "Downloading mpi-benchmarks from $osu_mpi_benchmark"
+	wget http://mvapich.cse.ohio-state.edu/download/mvapich/osu-micro-benchmarks-5.6.2.tar.gz
+	tar xvzf osu-micro-benchmarks-5.6.2.tar.gz
+	cd osu-micro-benchmarks-5.6.2
+	./configure --prefix=$(pwd) && make && make install
+	Verify_Result
+	LogMsg "OSU mpi-benchmarks  $osu_mpi_benchmark installation completed"
+	# set string to verify osu Benchmark is downloaded
+	osu_benchmark_bin=/usr/local/libexec/osu-micro-benchmarks/mpi/pt2pt/osu_latency
+	Verify_File $osu_benchmark_bin
+	cd HOMEDIR
```

Comment:
This is running under Sudo, so it goes to /root

Reply:
modified to move it to current directory before installation